src/scss/General.scss

```diff
@@ -2,7 +2,6 @@
 
 html, body {
   max-width: 100%;
-  overflow-x: hidden;
```

Comment:
This was breaking the programmatic smooth scrolling from Navbar links for some reason. Need to investigate why.